### PR TITLE
ipcache: print tunnel endpoint for RemoteEndpointInfo

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/7625)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: pkg/maps/ipcache/ipcache.go [hunk 1]
```diff
@@ -132,7 +132,7 @@ type RemoteEndpointInfo struct {
 }
 
 func (v *RemoteEndpointInfo) String() string {
-	return fmt.Sprintf("%d %d", v.SecurityIdentity, v.Key)
+	return fmt.Sprintf("%d %d %d", v.SecurityIdentity, v.Key, v.TunnelEndpoint)
```
Comment (joestringer) [inline pkg/maps/ipcache/ipcache.go]: Isn't the `TunnelEndpoint` an IP address? Seems like formatting it like that would be more readable.

Comment (aanm) [reply pkg/maps/ipcache/ipcache.go]: @joestringer it's a `[4]byte`, I could use
```
ip := net.IPv4(v.TunnelEndpoint[0],v.TunnelEndpoint[1],v.TunnelEndpoint[2],v.TunnelEndpoint[3])
ip.String()
```
will that work for you?

Comment (joestringer) [reply pkg/maps/ipcache/ipcache.go]: That sounds reasonable, so long as the byte-ordering is correct. I think it's in host byte-order anyway so this looks right to me.

Comment (aanm) [reply pkg/maps/ipcache/ipcache.go]: @joestringer I just noticed, is there any reason why `v.TunnelEndpoint` is not a `types.IPv4`?

Comment (joestringer) [reply pkg/maps/ipcache/ipcache.go]: I don't think there's a particular reason. `uint32` presumably just matched the C side, but so long as the value is the same size and alignment we shouldn't need to care about this.

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Review by joestringer — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

**Comment by aanm** [comment]
test-me-please

**Pull request merged**